### DISCUSSION TITLE
Goblin community Storage balance (Transparency)

### DISCUSSION THREAD

**Discussion by NoisyBucheron** [sunflower-land/sunflower-land#1229] [Ideas]
Hi guys,
With the new "Goblin community Storage" i got the idea that we should have a live balance of the Storage in a new "Storage" tab.
This will create a hype around the prizes for the next burns, giveaways etc... and at the same time show transparency about it.

What do you guys think about it?

![image](https://user-images.githubusercontent.com/104263513/178799597-802e3e0d-f3a9-486a-ba07-5c22f0833a20.png)

**Comment by Aminbtdds** [comment]
Great idea😍👏👏👏

**Comment by Yougottabekiddingme123** [comment]
Sound great. At the moment the community storage/pool just sounds abstract and players are angry because its so expensive to trade. If players can see the value in the community storage/pool that might calm people down. The giveaways should not be only for those who can sit through 2 hour discord or twitch streams. The giveaways should be in game and everyone should be able to enter it.

**Reply by adamhannigan** [reply]
Agreed. We will do some public giveaways shortly for this 😄

**Comment by adamhannigan** [comment]
@NoisyBucheron I love this idea. It would be great to keep tabs of how much the goblins currently own.

You can view it in Open Sea, but something similar in game would be nice 👌 

https://opensea.io/Goblin_Treasury

**Comment by j923sneaks** [comment]
I'll pick this up :)
